skynet/main.py

```diff
@@ -63,6 +62,8 @@ async def lifespan(main_app: FastAPI):
 
         await executor_shutdown()
 
+    await http_client.close()
```

Comment:
Just for my education here, why do we need this now?

Reply:
we don't really need it, it bugged me it was throwing an exception when i was closing the local instance

Reply:
Great, makes sense to fix it!